### PR TITLE
add LICENSE file

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
fixes the review issue on the cdb (https://content.minetest.net/packages/mt-mods/wrench/)

### CODE REVIEW

**Review by OgelGames — APPROVED** [review]

**Comment by wsor4035** [comment]
only has code license, missing media

**Review by S-S-X — CHANGES_REQUESTED** [review]
LGPLv2 seems fine but just mixing in CC-BY-SA isn't fine.

Not talking about license and wont comment that but it should make clear what parts are using CC-BY-SA and what parts are using LGPLv2.

While one is usually used for media and other for code it is not rule and currently this seems to be applying two different licenses for everything, that AFAIK cannot be done for LGPLv2 without either asking for permission from copyright holders or by using GPL licenses.

Basically just make it clear that CC is for media and LGPL is for code by adding two extra lines to LICENSE file.

**Comment by S-S-X** [comment]
Could also add 2 license files, one named `LICENSE` and other name `LICENSE-MEDIA`. GitHub should then be bale to also show multiple licenses for repository instead of displaying unknown / custom licnese.

**Review by S-S-X — APPROVED** [review]
Well, it is also on readme so I guess it is ok. For better visibility could however consider multiple license files instead of single combined file.

**Comment by OgelGames** [comment]
> Could also add 2 license files, one named `LICENSE` and other name `LICENSE-MEDIA`.

I like that idea.

**Comment by wsor4035** [comment]
if you want two files, the license file has to link to the other

**Comment by BuckarooBanzay** [comment]
> if you want two files, the license file has to link to the other

done, merging ~~to get this over with~~
